ios/FluentUI.Demo/FluentUI.Demo/ColoredPillBackgroundView.swift

```diff
@@ -37,10 +37,11 @@ class ColoredPillBackgroundView: UIView {
     func updateBackgroundColor() {
         switch style {
         case .neutral:
-            backgroundColor = UIColor(dynamicColor: fluentTheme.aliasTokens.colors[.background2])
+            backgroundColor = UIColor(dynamicColor: DynamicColor(light:  GlobalTokens.neutralColors(.grey98),
```

Comment:
There's an extra space between `light:` and `GlobalToken`, causing a linting error. I'm also getting a bunch of linting errors that xcode doesn't want to tell me about in some of my changes, so seems like we have something misconfigured.